pkgs/by-name/au/autotools-language-server/package.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, python3
+, fetchFromGitHub
+, fetchpatch
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "autotools-language-server";
+  version = "0.0.19";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "Freed-Wu";
+    repo = "autotools-language-server";
+    rev = version;
```

```suggestion
    rev = "refs/tags/${version}";
```
Help to avoid confusion.